### ISSUE TITLE
fs.watch can be called twice, so watchers[file].close() can crash

### ISSUE THREAD

**Issue by RandomEtc** [baryshev/ect#16]
I'm not 100% sure on this one yet. If it happens again I'll see if I can fix it.

**Comment by baryshev** [comment]
I'll try to check it.

**Comment by baryshev** [comment]
Can you check latest `0.5.7` release for this error?

**Comment by webuniverseio** [comment]
Figured out what to do to work around the error - cache should be set to `cache: true`, otherwise if I'll save parent layout - update the page, and then save child layout - on next update application will crash with:

```
TypeError: Cannot call method 'close' of undefined
at FSWatcher.<anonymous> (c:\dev\wpt\node_modules\ect\lib\ect.js:356:22)
at FSWatcher.EventEmitter.emit (events.js:107:17)
at FSEvent.FSWatcher._handle.onchange (fs.js:1016:12)
```

Thank you Vadim

**Comment by baryshev** [comment]
Fixed in `0.5.8`.
